### PR TITLE
Check for PublicKey is valid.

### PR DESCRIPTION
**Pull request by TheHmmka**
Just one more property to check if the PublicKey is valid or not.

### CODE REVIEW

**Review by bigearsenal — CHANGES_REQUESTED** [review]
Hi there, thank you for your contribution, just please re-add the extension in Package.swift. Thank you

**Comment by TheHmmka** [comment]
Done!

File: Package.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(name: "secp256k1", url: "https://github.com/Boilertalk/secp256k1.swift.git", from: "0.1.0"),
+        .package(name: "secp256k1", url: "https://github.com/v57/secp256k1.c.git", from: "0.1.0"),
```
Comment (bigearsenal) [inline Package.swift:22]: Hello. We can not verify this package right now, and the change is not related to the PR. Please reverse it.
The package secp256k1 will be verified in another PR

File: SolanaSwift.podspec [hunk 2]
```diff
@@ -40,7 +40,7 @@ Solana-blockchain client, written in pure swift, which supports keypair generati
 
   # s.public_header_files = 'Pod/Classes/**/*.h'
   # s.frameworks = 'UIKit', 'MapKit'
-  s.dependency 'secp256k1.swift', '~> 0.1.0'
+  s.dependency 'secp256k1.c', '~> 0.1.0'
```
Comment (bigearsenal) [inline SolanaSwift.podspec:43]: The same for this.

**Review by bigearsenal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**